.github/workflows/030_create_release.yaml

```diff
@@ -0,0 +1,93 @@
+name: Create release
+
+on:
+  push:
```

Comment:
I don't know if there is any difference with the code below but maybe the one below looks cleaner?
```
on:
  release:
    types: [published]
 ```

Reply:
On release will fire when a release is created manually, on push (with tag) will create a release for you. So the manual step is also automated :)

Reply:
Depends which flow you want to take but imo having releases and releasenotes created automatically is not always a good thing.
I prefer to make the release through the good github UI where you can still autogenerate releasenotes.
Releasenotes based on commits tend to be  a very long list which is hard to grab. If they are shorter and based on stories, features and bugs they are much more readable.
A manually created  release can also be marked as pre-release or as non-latest.
Creating a release through a tag means using the git cli or a git UI to place the tag instead of using the github create release UI.
Git tags are also an annoying thing sometimes: they are pulled to cloned repo's meaning that if you delete a release ( in case of an error or if something is forgotten) it can still be recreated if the tag is pushed back.

Reply:
I can split up the workflow to create an release candidate if the tag contains `rc`. For me personally if the initial draft is automated it can still be adjusted afterwards. I tend to sometimes forgot to create a release for a tag and then I notice it the next day..